requirements.txt

```diff
@@ -33,5 +33,5 @@ seaborn==0.10.1
 six==1.14.0
 toml==0.10.0
 typed-ast==1.4.1
-urllib3==1.26.5
```

Comment:
I want to lock CloudMapper to specific releases.

Reply:
Agree with you on this!
I've rebased the conflicts and the docker build works now with the `requirements.txt` from the `main` branch.